src/HMI/lux-data-bind.js

```diff
@@ -96,6 +96,73 @@ function isEqual(in1, in2) {
 
 }
 
+// Set a value deep within an arbitrary object
+LUX.setDeepObjectValue = function (obj, prop, value) {
+	//Note: This function was pulled out of Lux 
```

Comment:
Could you clarify the meaning of "this function was pulled out of LUX"? Maybe it was pulled from a separate file in the repo? I find it confusing since this still is in the LUX repo.

Reply:
there is a setDeepValue in Lux.js. I didn't want to modify that because i don't know if there are ramifications to the way it was functioning